client/assets/stylesheets/_nav-unification.scss

```
@@ -144,6 +145,30 @@
 	}
 
 	@include breakpoint-deprecated( '<480px' ) {
+		// New post button
+		.masterbar__item-new {
+			background: #32373c;
```

Or perhaps we should use `--color-sidebar-submenu-background` instead of hardcoding the color?

```suggestion
			background: var( --color-sidebar-submenu-background );
```